src/components/Burst.css

```diff
@@ -31,6 +31,16 @@
   text-align: center;
 }
 
+.Burst-dropdown {
+  height: 33px;
+  width: 114px;
+  margin-bottom: 15px;
+  margin-top: 5px;
+  border: 1px solid #000;
+  font-size: 1em;
+  outline: none !important;
```

Comment:
What was the reason behind this?

Reply:
this should eliminate the blue outline that appears on focus to follow the same styling as other gifsmos inputs, e.g. `.Burst-input` and `.Burst-button` screenshots related:
![Screen Shot 2019-06-21 at 11 42 43 AM](https://user-images.githubusercontent.com/32443210/59944534-d929cf80-9419-11e9-81b8-49a8708a03b8.png)
![Screen Shot 2019-06-21 at 11 42 58 AM](https://user-images.githubusercontent.com/32443210/59944542-df1fb080-9419-11e9-8619-51bf347ad8b5.png)